package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "If you are not familiar with linters, read [root level README](../README.md).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
- [ ] Nice job setting up the project, however, you forgot to add the `npm start` script alongside the `dev & build`.
so please add it and resubmit👍.